src/json.hpp.re2c

```diff
@@ -6168,6 +6168,21 @@ class basic_json
               thousands_sep(!loc->thousands_sep ? '\0' : loc->thousands_sep[0]),
               decimal_point(!loc->decimal_point ? '\0' : loc->decimal_point[0])
         {}
+        serializer(const serializer& cpy)
```

Comment:
I disagree, we should `= default` everything, since this is the actual behaviour.

Deleting them would potentially break code for no good reason.

I've read some comments about -Weff-c++ not being very relevant, (and I believe this warning is a bit too much).

But since this PR is really short, I think we can still merge it